### PR TITLE
Error message from requirements check on updatedb

### LINKED ISSUE #4405: updatedb - error message from requirements check
Running `drush updatedb` when a module is missing from the file system returns:

```
 [error]   (Currently using Missing or invalid module Array) 

 Requirements check reports errors. Do you wish to continue? (yes/no) [yes]:
```

The relevant code is in `src/Commands/core/UpdateDBCommands.php` - the updateCheckRequirements function. Has the $requirements array coming from update_check_requirements() changed format? 

```
$message .= ' (Currently using '. $requirement['title'] .' '. $requirement['value'] .')';
```

makes little sense when some of the (requirement) titles are

- Database support
- PHP memory limit
- File system
- Missing or invalid module

and values are various things (that might need to be rendered - like the error message for installed modules missing from the file system).

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 10.2.2 (dev)
| Drupal version? | 8.9.0
| PHP version     | 7.3
| OS?             | Linux

### PR DESCRIPTION
**Pull request by claudiu-cristea**
Fixes #4405

### CODE REVIEW

**Comment by claudiu-cristea** [comment]
Hm... this outputs something like:

```
[error]   (Currently using Incompatible modules The following modules are installed, but they are incompatible with PHP 7.1.33:<ul><li>Schema.org Article</li><li>Schema.org Event</li><li>Schema.org Metatag</li></ul>Review the <a href="https://www.drupal.org/docs/8/update/troubleshooting-database-updates"> suggestions for resolving this incompatibility</a> to repair your installation, and then re-run update.php.) 
```

Is there a way to transform HTML in console readable text?

File: src/Commands/core/UpdateDBCommands.php [hunk 1]
```diff
@@ -672,7 +673,12 @@ public function updateCheckRequirements()
                 if (isset($requirement['severity']) && $requirement['severity'] != REQUIREMENT_OK) {
                     $message = isset($requirement['description']) ? $requirement['description'] : '';
```
Comment (claudiu-cristea) [inline src/Commands/core/UpdateDBCommands.php:674]: Probably we need to pass 'description' as well?

Comment (weitzman) [reply src/Commands/core/UpdateDBCommands.php]: Sure, lets do that

Comment (claudiu-cristea) [reply src/Commands/core/UpdateDBCommands.php]: @weitzman already did :)

**Pull request merged**